packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -624,6 +624,17 @@ export const prodParasPolkadotCommon: EndpointOption[] = [
       color: '#e6777a',
       logo: 'fa;people-group'
     }
+  },
+  {
+    info: 'polkadotBridgeHub',
+    paraId: 1002,
+    providers: {
+      Parity: 'wss://polkadot-bridge-hub-rpc.polkadot.io'
+    },
+    text: 'BridgeHub',
+    ui: {
+      logo: nodesBridgeHubBlackSVG
```

Comment:
We should use the `nodesBridgeHub` logo here. The `nodesBridgeHubBlack` one is for Kusama.